build.gradle

```diff
@@ -5,11 +5,12 @@ plugins {
     id 'application'
     id 'jacoco'
 }
+application {
+    mainClass = 'seedu.address.Main'
+}
```

Comment:
@baskargopinath Some of these changes are not strictly needed for the Java 17 migration. If you think they are still useful, you can send separate PRs for them, with the justification.